### DISCUSSION TITLE
How does this compare to using Mise

### DISCUSSION THREAD

**Discussion by mvellandi** [webinstall/webi-installers#888] [General]
Hi, I just watched the intro video on the website, browsed the FAQ, and am curious how Webi compares to Mise.
I ask because I switched from Asdf to Mise around 4 mo. ago and am curious if there's specific benefits to Webi.
Thank-you!

**Comment by coolaj86** [comment]
That's a good question.

I haven't heard of mise before so I just took a quick look.

Webi:
- everything can be installed via URL
- has a cheat sheet for each tool
- uses simple symlinks for version switching
- uses POSIX shell scripts and PowerShell (i.e. for Docker and Windows)

Mise
- more tight shell integrations (version switching might not fully work in a limited POSIX shell)
- more tools that can be installed
- written in rust
- has someone in the group to make the website actually look nice

Mise appears to be a bit more sophisticated and integrated for a dev environment.

Webi appears to be just a tiny touch lighter and maybe more suitable for use in the deployed containers themselves, and native Windows (not just WSL)

As an actual user of Mise, what's your take?

**Reply by mvellandi** [reply]
So I just tried it out hearing it's "faster" than Asdf, and it is.
It feels like a drop-in replacement with a couple shortcut niceties (forgot which).
Well, I guess I'll have to go try Webi for a spin then!
